GeoQuizChallenge/app/src/test/java/com/bignerdranch/android/geoquiz/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.bignerdranch.android.geoquiz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
```

Comment:
this file is needed?

Reply:
I agree, I think you don't need this file